tests/test_videos.py

```diff
@@ -256,31 +215,6 @@ def test_convert_and_resize_ogv_works(self, low_res_ogv_video):
 
 
 
-""" *********** TEST SUBTITLES CONVERSION  *********** """
-
-class Test_subtitles_conversion(object):
-
-    def setup_method(self):
-        _clear_ricecookerfilecache()
-
-    def test_srt2vtt_works(self, srt_subtitles_file, srt_subtitles_file_strings):
-        subs_file = make_subtitles_file(srt_subtitles_file)
-        converted_subs_filename = subs_file.process_file()
-        subs_path = config.get_storage_path(converted_subs_filename)
-        assert os.path.exists(subs_path), 'no converted file found'
-        _, dotext = os.path.splitext(subs_path)
-        assert dotext == '.vtt', 'wrong extension of converted file'
-        with open(subs_path, 'r') as converted_file:
-            converted_contents = converted_file.read()
-            for string in srt_subtitles_file_strings:
-                assert string in converted_contents
-
-    def test_bad_subtitles_raises(self, bad_subtitles_file):
-        subs_file = make_subtitles_file(bad_subtitles_file)
-        pytest.raises(ValueError, subs_file.process_file)
```

Comment:
see `test_convertible_substitles_ar_srt` above for updated version of these tests